source/main/modules/window-manager/index.ts

```diff
@@ -219,6 +235,63 @@ export default class WindowManager {
       return
     }
 
+    this._mainWindow.on('show', async () => {
+      if (process.platform === 'darwin') {
+        if (this._tray == null) {
+          let basepath = app.getAppPath()
+          let image = await nativeImage.createThumbnailFromPath(basepath + '/source/main/assets/icons/black_white_128.png', { width: 16, height: 16 })
+          this._tray = new Tray(image)
+
+          // this._tray = new Tray(path.join(__dirname, 'assets/icons/128x128.png'))
+          const contextMenu = Menu.buildFromTemplate([
+            {
+              label: 'Show Zettlr',
+              click: () => {
+                this.showAnyWindow()
+              },
+              type: 'normal'
+            },
+            { label: '', type: 'separator' },
+            {
+              label: 'Quit',
+              click: () => {
+                app.quit()
+              },
+              type: 'normal'
+            }
+          ])
+          this._tray.setToolTip('This is Zettlr tray. \n click show Zettlr button to display app \n click quit to quit app')
+          this._tray.setContextMenu(contextMenu)
+        }
+      } else {
+        if (this._tray == null) {
+          this._tray = new Tray(path.join(__dirname, 'assets/icons/128x128.png'))
+          const contextMenu = Menu.buildFromTemplate([
+            {
+              label: 'Show Zettlr',
+              click: () => {
+                // Add show Zettlr window event to Windows
+                this.showMainWindow()
+              },
+              type: 'normal'
+            },
+            { label: '', type: 'separator' },
+            {
+              label: 'Quit',
+              click: () => {
+                // Add quit event to tray
+                // On Windows, left or right click the tray icon ➔ Quit will quit Zettlr. Same function as File ➔ Quit.
+                app.quit()
+              },
+              type: 'normal'
+            }
+          ])
+          this._tray.setToolTip('This is zettlr tray. \n click show Zettlr button to display app \n click quit to quit app')
```

Comment:
I suggest: `This is the Zettlr tray.` Upper case Z in Zettlr.
I suggest: `Select Show Zettlr to show the Zettlr app.` Use Select because the user may not click. The user may be using a touch device. Upper case S in Show to match the label `Show Zettlr`. Use the same verb 'show'.
I suggest: `Select Quit to quit the Zettlr app.`